perf/micro/current-thread-scheduler/operators/retry.js

```diff
@@ -0,0 +1,35 @@
+var RxOld = require('rx');
+var RxNew = require('../../../../index');
+
+module.exports = function (suite) {
+  var maxRetryCount = 25;
+  var oldRetryCount = 0;
+  var newRetryCount = 0;
+
+  var oldRetryWithCurrentThreadScheduler = RxOld.Observable.ofWithScheduler(RxOld.Scheduler.currentThread, 5)
```

Comment:
I've started just naming these `_old` and `_new` because the names are getting crazy in these files. haha.